packages/hint-doctype/README.md

```diff
@@ -0,0 +1,73 @@
+# Check that the DOCTYPE tag of the document is valid
+
+This hint checks if the HTML is using the most modern DOCTYPE.
+
+## Why is this important?
+
+In HTML, the doctype is the required "<!DOCTYPE html>" preamble
```

Comment:
Because this is  a quote from another website, we should put it as such. The aXe hint has an example on how we do this: https://github.com/webhintio/hint/blob/master/packages/hint-axe/README.md 
Can you mimic it?